src/librustc_mir/interpret/lvalue.rs

```diff
@@ -295,6 +295,8 @@ impl<'a, 'tcx> EvalContext<'a, 'tcx> {
 
             _ => bug!("field access on non-product type: {:?}", base_layout),
         };
+        //trace!("Field {} of {:?} is at offset {}{}", field_index, base_ty, offset.bytes(),
+        //    if packed { " (packed)" } else { "" });
```

Comment:
There's also a commented-out trace command somewhere in eval_context (in `write_value`). Basically these are traces that I consider too verbose to be present all the time when tracing, but have repeatedly needed when debugging things. Removing them makes it much more work to enable them again when needed; in particular, without these extra traces, it becomes hard to impossible to determine where an exception was raised (see #251), and then to figure out why it was raised. printf-style debugging ftw... ;)
What is the recommended way to deal with such tracing?